celerybeat.pid

```diff
@@ -0,0 +1 @@
+6
```

Comment:
@xuezhaokun Yes, this file is generated automatically by celery beat service for resource publication which is now part of HydroShare and when you rebuild hydroshare container, this file is regenerated, so make sure don't add them when you push to git.